generator-prisma-client/esbuild-cjs/README.md

```diff
@@ -0,0 +1,134 @@
+  - Your **Prisma Postgres direct TCP connection string** which you will use with Prisma Client.
+    Learn more in the [docs](https://www.prisma.io/docs/postgres/database/direct-connections).
+
+## Tech Stack
+
+- Next.js 15
+  - Runtime: Node.js 20.19.0
+  - Bundler: Turbopack (stable for `dev`, alpha for `build`)
+- CJS
+  - `package.json` contains `{ "type": "commonjs" }`
+  - `esbuild` specifies `format: 'cjs'`
+- Prisma Client with the `prisma-client` generator
+  See the [Prisma schema file](./prisma/schema.prisma) for details.
+  
+  ```prisma
+  generator client {
+    provider = "prisma-client"
+    output = "../src/generated/prisma"
+    previewFeatures = ["driverAdapters", "queryCompiler"]
+    moduleFormat = "cjs"
+  }
+
+  datasource db {
+    provider  = "postgresql"
+    url       = env("DATABASE_URL")
+    directUrl = env("DIRECT_URL")
+  }
+  ```
+
+## Getting started
+
+### 1. Clone the repository
+
+Clone the repository, navigate into it and install dependencies:
+
+```
+git clone git@github.com:prisma/prisma-examples.git --depth=1
+cd prisma-examples/generator-prisma-client/esbuild-cjs
+pnpm install
+```
+
+### 2. Configure environment variables
+
+Create a `.env` in the root of the project directory:
+
+```bash
+touch .env
+```
+
+Now, open the `.env` file and set the `DATABASE_URL` environment variables with the values of your connection string and your Prisma Postgres connection string:
+
+```bash
+# .env
+
+# Prisma Postgres connection string (used for migrations)
+DATABASE_URL="__YOUR_PRISMA_POSTGRES_CONNECTION_STRING__"
+
+# Postgres connection string (used for queries by Prisma Client)
+DIRECT_URL="__YOUR_PRISMA_POSTGRES_DIRECT_CONNECTION_STRING__"
+
+NEXT_PUBLIC_URL="http://localhost:3000"
+```
+
+Note that `__YOUR_PRISMA_POSTGRES_CONNECTION_STRING__` is a placeholder value that you need to replace with the values of your Prisma Postgres + Accelerate connection string. Notice that the Accelerate connection string has the following structure: `prisma+postgres://accelerate.prisma-data.net/?api_key=<api_key_value>`.
+
+Note that `__YOUR_PRISMA_POSTGRES_DIRECT_CONNECTION_STRING__` is a placeholder value that you need to replace with the values of your Prisma Postgres direct TCP connection string. The direct connection string has the following structure: `postgres://<username>:<password>@<host>:<port>/<database>`.
+
+### 3. Run a migration to create the database structure and seed the database
+
+The [Prisma schema file](./prisma/schema.prisma) contains a single `Quotes` model and a `QuoteKind` enum. You can map this model to the database and create the corresponding `Quotes` table using the following command:
+
+```
+pnpm prisma migrate dev --name init
+```
+
+You now have an empty `Quotes` table in your database. Next, run the [seed script](./prisma/seed.ts) to create some sample records in the table:
```

Comment:
I think you missed adding `seed.ts` file. Nothing would happen as the file is missing.